csharp/src/Google.Protobuf.Test/Reflection/CustomOptionsTest.cs

```diff
@@ -184,13 +184,21 @@ public void NoOptions()
             var fileDescriptor = UnittestProto3Reflection.Descriptor;
             var messageDescriptor = TestAllTypes.Descriptor;
             Assert.NotNull(fileDescriptor.CustomOptions);
+            Assert.DoesNotThrow(() => { fileDescriptor.GetOption(Fileopt); });
```

Comment:
What value does `Assert.DoesNotThrow` add? A thrown error would fail the test anyway.

Wouldn't a better check be that these methods return null if there is no option, and not null if there is.

Reply:
`Assert.DoesNotThrow` is a style that's been enforced in previous PRs. Though, yes, a better check would be to make sure GetOption returns the default value of the extension.